Cargo.toml

```diff
@@ -2,8 +2,8 @@
 name = "gosub-engine"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.73"
 authors = ["Joshua Thijssen <jaytaph@deadcode.nl>"]
-
 description = "html5 browser engine"
 license = "MIT"
 repository = "https://github.com/jaytaph/gosub-browser"
```

Comment:
I don't know how significant it is, but this repo URL is now outdated

Reply:
Good catch.